tests/test_action.py

```diff
@@ -231,7 +231,7 @@ def test_non_unicode_string_error_replace(self):
         capture = StringIO()
         my_action._print_process_output(Mock(), not_unicode, capture, realtime)
         # get the replacement char
-        expected = '�' if six.PY3 else unicode('�', 'utf-8')
+        expected = '�' if six.PY3 else '�'.decode('utf-8')
```

Comment:
if you run pyflakes with `doit pyflakes` it handles this problem on python3. But this way is nicer :)